backup/queries_relations.go

```diff
@@ -126,10 +126,22 @@ func getUserTableRelations(connectionPool *dbconn.DBConn) []Relation {
 		WHERE e.reloid IS NULL)`
 	}
 
+	prtPages := "prt.pages"
+	prt := `LEFT JOIN (
+		SELECT
+			p.parrelid,
+			sum(pc.relpages) AS pages
+		FROM pg_partition_rule AS pr
+		JOIN pg_partition AS p ON pr.paroid = p.oid
+		JOIN pg_class AS pc ON pr.parchildrelid = pc.oid
+		GROUP BY p.parrelid
+	) AS prt ON prt.parrelid = c.oid`
 	// In GPDB 7+, root partitions are marked as relkind 'p'.
```

Comment:
maybe move the comment inside if? since it only applies to line 142. the same thing is in the getUserTableRelationsWithIncludeFiltering function.

Reply:
this comment is not touched by this patch

Reply:
Yes, but I think it might make sense to move it. since before, only `relkindFilter` was changed inside if.

Reply:
I don't see any sense in increasing diff because of such small corrections